mne/channels/tests/test_interpolation.py

```diff
@@ -18,66 +17,85 @@
 
 raw = io.Raw(raw_fname, add_eeg_ref=False)
 events = read_events(event_name)
-picks = pick_types(raw.info, meg=False, eeg=True, exclude=[])
-
-reject = dict(eeg=80e-6)
+picks_eeg = pick_types(raw.info, meg=False, eeg=True, exclude=[])
+picks_meg = pick_types(raw.info, meg=True, eeg=False, exclude=[])
+picks = pick_types(raw.info, meg=True, eeg=True, exclude=[])
+
+epochs_eeg = Epochs(raw, events, event_id, tmin, tmax, picks=picks_eeg,
+                    preload=True, reject=dict(eeg=80e-6))
+epochs_meg = Epochs(raw, events, event_id, tmin, tmax, picks=picks_meg,
+                    preload=True, reject=dict(grad=1000e-12, mag=4e-12))
```

Comment:
I know it wasn't this way before, but could you nest this in a function call? At one point we had a problem where running `nosetests` took up way too much memory, because all of the `test_` modules get loaded on init, and we had a ton of things like this where objects get created during initialization. Putting these into a function costs a little bit more in CPU and HDD time, but will save in memory because they get loaded (and un-loaded!) on-demand.